revproxy/views.py

```diff
@@ -38,30 +45,57 @@ def __init__(self, *args, **kwargs):
 
     @property
     def upstream(self):
+        """This method is not implemanted on the base class """
         raise NotImplementedError('Upstream server must be set')
 
     @property
     def diazo_rules(self):
+        """Method to create a path to the file containing the diazo rules
+
+        :returns: Path of diazo rules file
+        """
         child_class_file = sys.modules[self.__module__].__file__
         app_path = os.path.abspath(os.path.dirname(child_class_file))
         return os.path.join(app_path, 'diazo.xml')
 
     @classonlymethod
     def as_view(cls, **initkwargs):
```

Comment:
as_view always return a callable view so we can just document the behavior we added.